dashboard/src/components/userSettings/UserSettingsDialog.tsx

```diff
@@ -40,6 +41,78 @@ interface UserSettingsTabConfig {
 
 export default function UserSettingsDialog({ open, onOpenChange }: UserSettingsDialogProps) {
   const { settings, isLoading, isSaving, error, saveSettings } = useUserSettings();
+  const tDialog = useTranslations('components.userSettings.dialog');
+
+  if (isLoading) {
+    return (
+      <Dialog open={open} onOpenChange={onOpenChange}>
+        <DialogContent className='sm:max-w-[800px] md:min-w-[700px] lg:min-w-[900px]'>
+          <div className='flex flex-col items-center justify-center space-y-3 py-16'>
+            <Spinner />
+            <p className='text-muted-foreground text-sm'>{tDialog('loading')}</p>
+          </div>
+        </DialogContent>
+      </Dialog>
+    );
+  }
+
+  if (error) {
+    return (
+      <Dialog open={open} onOpenChange={onOpenChange}>
+        <DialogContent className='sm:max-w-[800px] md:min-w-[700px] lg:min-w-[900px]'>
+          <div className='flex flex-col items-center justify-center space-y-3 py-16'>
+            <AlertTriangle className='text-destructive h-8 w-8' />
+            <div className='text-center'>
+              <p className='text-destructive font-medium'>{tDialog('loadFailed')}</p>
+              <p className='text-muted-foreground mt-1 text-sm'>{error}</p>
+            </div>
+          </div>
+        </DialogContent>
+      </Dialog>
+    );
+  }
+
+  if (!settings) {
+    return (
+      <Dialog open={open} onOpenChange={onOpenChange}>
+        <DialogContent className='sm:max-w-[800px] md:min-w-[700px] lg:min-w-[900px]'>
+          <div className='flex items-center justify-center py-8'>
+            <span>{tDialog('noSettings')}</span>
+          </div>
+        </DialogContent>
+      </Dialog>
+    );
+  }
```

Comment:
I get this error from these:
<img width="663" height="98" alt="image" src="https://github.com/user-attachments/assets/2d050533-0b2a-480f-8679-16e4cce2b31e" />

I think they just need to have a:
`<DialogTitle>`

As the message says, you can wrap them in a `VisuallyHidden`, such that they're not visible 😄